### PR TITLE
Prepare v1.5.0 release

### PR DESCRIPTION
**Pull request by ThomsonTan**
## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,11 +15,32 @@ Increment the:
...
 * [EXAMPLE] Fix memory ownership of InMemorySpanExporter (#1473) [#1471](https://github.com/open-telemetry/opentelemetry-cpp/pull/1471)
-* [EXPORTER] Add metrics OTLP/HTTP exporter [#1487](https://github.com/open-telemetry/opentelemetry-cpp/pull/1487)
+* [EXPORTER TESTS] Prometheus unit test [#1461](https://github.com/open-telemetry/opentelemetry-cpp/pull/1461)
+* [DOCS] Update docs to use relative code links. [#1447](https://github.com/open-telemetry/opentelemetry-cpp/pull/1447)
+* [SDK] Remove reference to deprecated InstrumentationLibrary in OTLP [#1469](https://github.com/open-telemetry/opentelemetry-cpp/pull/1469)
+* [SDK] Fix trace kIsSampled flag set incorrectly [#1465](https://github.com/open-telemetry/opentelemetry-cpp/pull/1465)
+* [METRICS EXPORTER] OTLP gRPC Metrics Exporter [#1454](https://github.com/open-telemetry/opentelemetry-cpp/pull/1454)
+* [EXPORTER] fix prometheus exporter failure type [#1460](https://github.com/open-telemetry/opentelemetry-cpp/pull/1460)
 * [EXPORTER] OTLP http exporter allow concurrency session ([#1209](https://github.com/open-telemetry/opentelemetry-cpp/pull/1209))
```
Comment (lalitb) [inline CHANGELOG.md:42]: Can we remove this PR, as it was not merged?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,11 +15,32 @@ Increment the:
...
-* [EXT] `curl::HttpClient` use `curl_multi_handle` instead of creating a thread
+* [SDK] Fix build warnings about missing aggregates. [#1368](https://github.com/open-telemetry/opentelemetry-cpp/pull/1368)
```
Comment (lalitb) [inline CHANGELOG.md:43]: Some mixup in this line. Can you relook once again? Thanks.

**Review by lalitb — APPROVED** [review]
LGTM, once comments are fixed.

**Comment by lalitb** [comment]
Oh lots of tests failure. Some issue it seems.

**Comment by ThomsonTan** [comment]
> Oh lots of tests failure. Some issue it seems.

Looking.

**Comment by lalitb** [comment]
> > Oh lots of tests failure. Some issue it seems.
> 
> Looking.

Not sure if this is the reason, but `api/include/opentelemetry/version.h` needs to be updated too.

File: CHANGELOG.md (hunk 1 above)
Comment (ThomsonTan) [reply CHANGELOG.md:42]: Good catch, removed it.

File: CHANGELOG.md (hunk 2 above)
Comment (ThomsonTan) [reply CHANGELOG.md:42]: Fixed the mix, thanks!

**Comment by owent** [comment]
Hopes v1.5.0 can include #1507 , so that all OTLP exporters will use the new `scope_*` field.Or only the logs and traces use the new  one but metrics use the old one.
> The old OTLP fields are already removed in the main branch of opentelemetry-proto.

**Comment by esigo** [comment]
> Hopes v1.5.0 can include #1507 , so that all OTLP exporters will use the new `scope_*` field.Or only the logs and traces use the new  one but metrics use the old one.
> > The old OTLP fields are already removed in the main branch of opentelemetry-proto.


@ThomsonTan @lalitb I agree with @owent, it would be great if we could include #1507 in this release.

**Review by esigo — APPROVED** [review]
LGTM
Thanks for the PR :)

**Review by owent — APPROVED** [review]

**Comment by ThomsonTan** [comment]
> > Hopes v1.5.0 can include #1507 , so that all OTLP exporters will use the new `scope_*` field.Or only the logs and traces use the new  one but metrics use the old one.
> > > The old OTLP fields are already removed in the main branch of opentelemetry-proto.
> 
> @ThomsonTan @lalitb I agree with @owent, it would be great if we could include #1507 in this release.

Thanks for the suggestion, #1507 will be included in v1.5.0:)

**Pull request merged**